src/crypto/CHIPCryptoPALmbedTLS.cpp

```diff
@@ -981,8 +980,7 @@ CHIP_ERROR Spake2p_P256_SHA256_HKDF_HMAC::InitInternal(void)
     result = mbedtls_ecp_group_load(&context->curve, MBEDTLS_ECP_DP_SECP256R1);
     VerifyOrExit(result == 0, error = CHIP_ERROR_INTERNAL);
 
-    context->md_info = mbedtls_md_info_from_type(MBEDTLS_MD_SHA256);
```

Comment:
Why do we not need to set `context->md_info` anymore?

Reply:
It's not referenced anywhere in the codebase. I consider removing it a general cleanup action which I did when I noticed it was taking up needless space in the context structure.

I can drop this change from the PR if it's controversial, but again, it's not referenced anywhere in the codebase that I can find.

Reply:
Please file a followup to remove it, if it's not used?  Leaving it unassigned but there as an attractive nuisance seems like a bad idea.

Reply:
I'm [removing it in this PR](https://github.com/project-chip/connectedhomeip/pull/21415/commits/30483087b205ad9e694353fa04fe3fc368b9bbf2) already. Are you saying you want the change in a separate PR?

Reply:
No, I just missed that the member as a whole is being removed.  My apologies!